src/modules/facility-activation-transaction/dto/create-facility-activation-transaction-response.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { EXAMPLES } from '@ukef/constants';
+import { AcbsBundleId } from '@ukef/helpers';
+
+export class CreateFacilityActivationTransactionResponse {
+  @ApiProperty({
+    readOnly: true,
+    minLength: 10,
+    maxLength: 10,
```

Comment:
Will `length` decorator suffice both `minLength` and `maxLength`?

Reply:
Decorator ApiProperty is coming from package @nestjs/swagger and this decorator **doesn't** support length.

OpenAPI spec also **doesn't** mention length, just min and max for strings. https://swagger.io/docs/specification/data-models/data-types/